Directory.Build.targets

```diff
@@ -125,6 +125,11 @@
 
   <Import Project="$(ToolsDir)/Build.Common.targets" Condition="Exists('$(ToolsDir)/Build.Common.targets')" />
 
+  <!-- Import Arcade targets -->
+<!--  <Import Project="$(PackagesDir)/$(GenAPIPackage.ToLower())/$(GenAPIPackageVersion)/build/$(GenAPIPackage).targets" /> -->
```

Comment:
You need to add the notsupportes support to fix the openssl issue.

Reply:
@weshaggard you think it's an ordering issue? In the current state we're importing notsupported.targets inside of build.common.targets, while the other 2 are coming from here